justfile

```diff
@@ -0,0 +1,29 @@
+set quiet
+
+import? '../sdk-codegen/justfile'
+
+_default:
+    just --list
+
+test no_build="" config="Debug":
+    dotnet test {{no_build}} -f net8.0 src/StripeTests/StripeTests.csproj -c {{config}}
+
+# skip build and test in release mode
+test-release: (test "--no-build" "Release")
+alias ci-test := test-release
```

Comment:
I added an alias because CI calls this and it's nice to be explicit about which recipe CI will be using. But we could just as easily amend the GH actions to call `test-release` if we preferred.